applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp

```diff
@@ -183,7 +183,7 @@ void EmbeddedSkinVisualizationProcess::ComputeNewNodesInterpolation(){
     const int n_new_elems = mNewElementsPointers.size();
     ModelPart::ElementIterator new_elems_begin = mNewElementsPointers.begin();
 
-    #pragma omp parallel for
+    //#pragma omp parallel for
```

Comment:
i don't see an error here...

Reply:
Me neither. But Valgrind complains about that...